### PR TITLE
EIP-5192: add event LockingStatusChanged(uint256 tokenId, bool status)

### PR DESCRIPTION
**Pull request by 0xanders**
Add `event LockingStatusChanged(uint256 tokenId, bool status)`

So the locking status of the NFT could be updated  off-chain.

### CODE REVIEW

File: EIPS/eip-5192.md [hunk 1]
```diff
@@ -34,6 +34,9 @@ A token with a `uint256 tokenId` may be inseparably-nound to a receiving account
 pragma solidity ^0.8.0;
 
 interface IERC5192 {
+  /// @notice Emitted when locking status is changed
+  event UpdateLockingStatus(uint256 tokenId, bool status);
```
Comment (xinbenlv) [inline EIPS/eip-5192.md:38]: [Optional] suggest 
```suggestion
  event LockinStatusChanged(uint256 tokenId, bool status);
```

**Review by xinbenlv — APPROVED** [review]
Reasonable updates: add an event interface, adding an author

**Comment by xinbenlv** [comment]
Typo :p

File: EIPS/eip-5192.md [hunk 2]
```diff
@@ -34,6 +34,9 @@ A token with a `uint256 tokenId` may be inseparably-nound to a receiving account
 pragma solidity ^0.8.0;
 
 interface IERC5192 {
+  /// @notice Emitted when locking status is changed
```
Comment (TimDaub) [inline EIPS/eip-5192.md:37]: I think we should add an `@dev` note on what should happen upon minting. E.g. when minting "a regular NFT", what should `bool status` be?

File: EIPS/eip-5192.md [hunk 3]
```diff
@@ -34,6 +34,9 @@ A token with a `uint256 tokenId` may be inseparably-nound to a receiving account
...
+  event LockingStatusChanged(uint256 tokenId, bool status);
```
Comment (TimDaub) [inline EIPS/eip-5192.md:38]: I'm unsure if I like the readability of `LockingStatusChanged(..., bool status)`.

Naturally, what does it mean `Locking Status Changed to false`? If we reflect upon it actively, then we can probably understand that a lock is locked on `true` and open on `false.`

What about:

`event Locked(uint256 tokenId, bool state)` where if `state=true` then the token is `Locked` and if `state=false` the token is NOT `Locked`.

**Review by TimDaub — CHANGES_REQUESTED** [review]
left few more comments

File: EIPS/eip-5192.md (hunk 3 above)
Comment (0xanders) [reply EIPS/eip-5192.md:38]: How about these two events:  Locked and Unlocked

/// @notice Emitted when the locking status is changed to locked
/// @param tokenId The identifier for a token.
event Locked(uint256 tokenId);

/// @notice Emitted when the locking status is changed to unlocked
/// @param tokenId The identifier for a token.
event Unlocked(uint256 tokenId);

@TimDaub

Comment (TimDaub) [reply EIPS/eip-5192.md:38]: ok. But also we must decide what should be done if e.g. a token is minted. Must a `Unlocked(tokenId)` event be emitted? Whatever we decide on, let's please also add it to the natspec using `/// @dev`.

**Pull request merged**